src/dependencies.jl

```diff
@@ -1043,13 +1043,8 @@ using SHA
 function sha_check(path, sha)
     open(path) do f
         calc_sha = sha256(f)
-	# Workaround for SHA.jl API change.  Safe to remove once SHA versions
-	# < v0.2.0 are rare, e.g. when Julia v0.4 is deprecated.
```

Comment:
should set a minimum version for SHA if we do this, but better to leave the flexibility